### PR TITLE
Add persistent storage to Eddystone demo

### PR DESCRIPTION
**Pull request by andresag01**
@rgrover @jaustin

### CODE REVIEW

File: BLE_EddystoneService/source/EddystoneService.cpp [hunk 1]
```diff
@@ -89,7 +89,7 @@ void EddystoneService::setURLData(const char *urlDataIn)
     urlFrame.setURLData(urlDataIn);
 }
 
-void EddystoneService::setUIDData(const UIDNamespaceID_t *uidNamespaceIDIn, const UIDInstanceID_t *uidInstanceIDIn)
+void EddystoneService::setUIDData(const UIDNamespaceID_t &uidNamespaceIDIn, const UIDInstanceID_t &uidInstanceIDIn)
```
Comment (jaustin) [inline BLE_EddystoneService/source/EddystoneService.cpp]: I'd like to see this change made as a separate commit, as I don't think it is directly related to the ConfigPrams stuff directly, right?

Comment (rgrover) [inline BLE_EddystoneService/source/EddystoneService.cpp]: yes, I agree. this should be separated.

File: BLE_EddystoneService/source/EddystoneService.h [hunk 2]
```diff
@@ -112,7 +112,7 @@ class EddystoneService
 
     void setURLData(const char *urlDataIn);
 
-    void setUIDData(const UIDNamespaceID_t *uidNamespaceIDIn, const UIDInstanceID_t *uidInstanceIDIn);
+    void setUIDData(const UIDNamespaceID_t &uidNamespaceIDIn, const UIDInstanceID_t &uidInstanceIDIn);
```
Comment (jaustin) [inline BLE_EddystoneService/source/EddystoneService.h]: ps, "git add -p" is definitely your friend :D

http://johnkary.net/blog/git-add-p-the-most-powerful-git-feature-youre-not-using-yet/

**Comment by andresag01** [comment]
Closing pull request to separate changes into multiple commits.

**Pull request closed without merging**